### DISCUSSION TITLE
Controlling the TOTCODE in .csv for pivot table files

### DISCUSSION THREAD

**Discussion by bjanez** [sdcTools/tauargus#11] [Q&A]
In older versions of Tau Argus, for example 3.5.0 or 4.1.4, the default TOTCODE in the .csv for pivot table file output was always 'Total' no matter what was specified in the .rda for that variable as the TOTCODE. In the newer versions (I don't know from which version onward) of Tau Argus, this behavior was changed in a way that now TOTCODE in the .csv was the same as specified for that variable in the corresponding .rda file. I totally understand this change in .csv output and probably it makes sense for most of the users, but due to some legacy code in our procedures which is based on older Tau Argus versions, I would like to ask if I can somehow modify or change the option somewhere so that even in the newest Tau Argus (4.2.4b), the TOTCODE in the .csv will always be 'Total' no matter what is specified in the .rda. Basically, I would like to get the same behavior for CSV for pivot table outputs as you get when using 3.5.0 or 4.1.4 versions. Is that possible? Would it be possible to add this option in future versions of Tau Argus?

Kind Regards, 

Janez Bijec

**Comment by ppdewolf** [comment]
At the moment it is not possible to change this behaviour. For most users the desired behaviour is to keep the `TOTCODE` as specified in the `.rda`-file. See also [issue #77](https://github.com/sdcTools/UserSupport/issues/77).
